src/display/font_loader.js

```diff
@@ -77,6 +77,25 @@ class BaseFontLoader {
     }
   }
 
+  async loadNonEmbeddedFont(loadedName, localNames) {
+    if (!this.isFontLoadingAPISupported) {
+      return false;
+    }
+
+    if (!Array.isArray(localNames)) {
+      localNames = [localNames];
+    }
+    localNames = localNames.map(name => `local('${name}')`).join(" ");
```

Comment:
If there are multiple local names to try, I believe the `local('...')` entries should be separated by commas, not just spaces.